framework/packages/abstract-testing/src/lib.rs

```diff
@@ -191,6 +196,21 @@ pub mod module {
     pub const TEST_MODULE_RESPONSE: &str = "test_module_response";
 }
 
+pub mod mock_bech32 {
+    use abstract_std::native_addrs::TEST_ABSTRACT_CREATOR;
+    use cosmwasm_std::{Addr, Api, CanonicalAddr};
+    use cw_orch::mock::MockBech32;
+
+    pub fn mock_bech32_admin(bech32: &MockBech32) -> Addr {
+        bech32
+            .app
+            .borrow()
+            .api()
+            .addr_humanize(&CanonicalAddr::from(TEST_ABSTRACT_CREATOR))
+            .unwrap()
+    }
+}
+
```

Comment:
A bit annoying to refactor, maybe annoying to use? But we need predetermined sender

Reply:
Can't we set the sender when deploying the contracts only?

Reply:
I.e. in `deploy_on`

Reply:
Would this be better as a trait?

Reply:
> Would this be better as a trait?

Not sure what to implement it on. For Abstract builder we can add `build_test()` for MockBech32 Env without sender param

Reply:
On `MockBase<A, S>`? 

But probably easier to have a `deploy_on_mock` fn? Would be nice to stick to the `deploy_on` naming to stay consistent.